### PR TITLE
DOC: Edits to Ubuntu SOS; changed SOS/UOS to Service VM/User VM

### PR DESCRIPTION
**Pull request by deb-intel**
Signed-off-by: Deb Taylor <deb.taylor@intel.com>

### CODE REVIEW

**Comment by deb-intel** [comment]
@gvancuts Please make sure I have correctly changed references to SOS and UOS. Consider that I might have made changes that should remain as OS. Should the title be updated to "Using Ubuntu as the Service VM"?

File: doc/tutorials/using_ubuntu_as_sos.rst [hunk 1]
```diff
@@ -3,17 +3,17 @@
 Using Ubuntu as the Service OS
```
Comment (deb-intel) [inline doc/tutorials/using_ubuntu_as_sos.rst]: Should the title be updated to "Service VM"?

File: doc/tutorials/using_ubuntu_as_sos.rst [hunk 2]
```diff
@@ -182,51 +180,44 @@ You can download latest Service OS kernel from
...
 
         sudo update-grub
 
-#. Reboot the system
+#. Reboot the system.
 
-   Reboot system. You should see the Grub menu with the new "ACRN ubuntu SOS"
-   entry. Select it and proceed to booting the platform. The system will start
-   the Ubuntu Desktop and you can now log in (as before).
+   Reboot the system. You should see the Grub menu with the new "ACRN ubuntu SOS" entry. Select it and proceed to booting the platform. The system will start the Ubuntu Desktop and you can now log in (as before).
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_sos.rst]: Let's change it here too actually (and you'll need to make a similar adjustment to the line above that shows the `menuentry`:
```suggestion
   Reboot the system. You should see the Grub menu with the new "ACRN Ubuntu Service VM" entry. Select it and proceed to booting the platform. The system will start the Ubuntu Desktop and you can now log in (as before).
```

File: doc/tutorials/using_ubuntu_as_sos.rst (hunk 1 above)
Comment (gvancuts) [reply doc/tutorials/using_ubuntu_as_sos.rst]: Let's make that: "Running Ubuntu in the Service VM". Would that work for you?

Comment (deb-intel) [reply doc/tutorials/using_ubuntu_as_sos.rst]: Yes! I think this is much better and more accurate. Thanks.

File: doc/tutorials/using_ubuntu_as_sos.rst [hunk 3]
```diff
@@ -182,51 +180,44 @@ You can download latest Service OS kernel from
                 linux  /boot/acrn/org.clearlinux.iot-lts2018-sos.4.19.78-98  pci_devices_ignore=(0:18:1) console=tty0 console=ttyS0 root=PARTUUID=<UUID of rootfs partition> rw rootwait ignore_loglevel no_timer_check consoleblank=0 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_gvt=1 i915.enable_guc=0 hvlog=2M@0x1FE00000
         }
 
-   .. note::
-        You need to adjust this to use your partition UUID (``PARTUUID``) for
-        the ``root=`` parameter (or use the device node directly).
+     .. note::
+          Adjust this to use your partition UUID (``PARTUUID``) for the ``root=`` parameter (or use the device node directly).
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_sos.rst]: The alignment seems off here, or is it me?

File: doc/tutorials/using_ubuntu_as_sos.rst [hunk 4]
```diff
@@ -182,51 +180,44 @@ You can download latest Service OS kernel from
...
-   Reboot system. You should see the Grub menu with the new "ACRN ubuntu SOS"
-   entry. Select it and proceed to booting the platform. The system will start
-   the Ubuntu Desktop and you can now log in (as before).
+   Reboot the system. You should see the Grub menu with the new "ACRN ubuntu SOS" entry. Select it and proceed to booting the platform. The system will start the Ubuntu Desktop and you can now log in (as before).
 
    .. note::
-       If you don't see the Grub menu after rebooting the system (and you're
-       not booting into the ACRN hypervisor), you'll need to enter the
-       EFI firmware at boot (using :kbd:`F10`) and manually select ``ACRN Hypervisor``.
+       If you don't see the Grub menu after rebooting the system (and you are
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_sos.rst]: Same here, is the alignment correct?

File: doc/tutorials/using_ubuntu_as_sos.rst (hunk 3 above)
Comment (deb-intel) [reply doc/tutorials/using_ubuntu_as_sos.rst]: @gvancuts I made a few more updates and they aligned correctly when I viewed them locally. I'll check again.

File: doc/tutorials/using_ubuntu_as_sos.rst (hunk 4 above)
Comment (deb-intel) [reply doc/tutorials/using_ubuntu_as_sos.rst]: Just checked. It looks correct.

**Pull request merged**